services/api-dotnet/src/app/Api.Sql/Program.cs

```diff
@@ -32,7 +34,14 @@
 services.AddApiControllers();
 services.AddSwagger();
 services.AddHttpContextAccessor();
-services.AddAuthorization();
+services
+    .AddAuthentication(Constants.AuthenticationScheme)
+    .AddScheme<TokenAuthenticationSchemeOptions, TokenAuthenticationHandler>(Constants.AuthenticationScheme, _ => { });
```

Comment:
The last parameter is nullable. It seems we can use `null` (or `default`) instead of empty lambda